torch/_inductor/codegen/triton.py

```diff
@@ -2379,10 +2379,18 @@ def codegen_node_schedule(self, node_schedule, numel, reduction_numel):
 
         self.codegen_node_schedule_with_kernel(node_schedule, kernel)
 
-        src_code = kernel.codegen_kernel()
+        with V.set_kernel_handler(kernel):
```

Comment:
could we combine this with the set kernel handler line below?

Reply:
yea, merged locally. Will send out a update after I look into the test failures.